src/ucp/rndv/rndv_get.c

```diff
@@ -140,19 +149,11 @@ ucp_proto_rndv_ats_init(const ucp_proto_init_params_t *params)
     return UCS_OK;
 }
 
-static ucs_status_t ucp_proto_rndv_ats_progress(uct_pending_req_t *self)
-{
-    ucp_request_t *req = ucs_container_of(self, ucp_request_t, send.uct);
-
-    return ucp_proto_rndv_ack_progress(
-            req, UCP_AM_ID_RNDV_ATS, ucp_proto_request_zcopy_complete_success);
-}
-
 static ucp_proto_t ucp_rndv_ats_proto = {
     .name       = "rndv/ats",
     .flags      = 0,
     .init       = ucp_proto_rndv_ats_init,
     .config_str = ucp_proto_rndv_ack_config_str,
-    .progress   = ucp_proto_rndv_ats_progress
+    .progress   = {ucp_proto_rndv_get_common_ats_progress}
 };
```

Comment:
do we really need a separate proto for ATS now?

Reply:
yes: zcopy protocols are currently not supporting 0-length message